ckanext/blob_storage/plugin.py

```diff
@@ -139,10 +143,31 @@ def register_authz_bindings(self, authorizer):
         authorizer.register_authorizer('obj', authz.check_object_permissions,
                                        actions={'update', 'read'},
                                        subscopes=(None, 'data', 'metadata'))
+
+        authorizer.register_authorizer('obj', helpers.check_resource_permissions,
+                                       actions={'update', 'read'},
+                                       subscopes=(None, 'data', 'metadata'))
+
         authorizer.register_action_alias('write', 'update', 'obj')
         authorizer.register_scope_normalizer('obj', authz.normalize_object_scope)
 
-    # IResourceDownloadHandler
-
     def resource_download(self, resource, package, filename=None, inline=False, activity_id=None):
         return download_handler(resource, package, filename, inline, activity_id)
+
+
+def _all_entity_actions(entity_checks):
```

Comment:
looks like unused function? Also it's best not to create too much helpers inside plugin.py. If you need some create them in modules and import here in plugin.py.